libs/coin-modules/coin-cosmos/src/bridge/js.ts

```diff
@@ -0,0 +1,128 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
```

Comment:
It could be nice to already rename this file `index.ts`

Reply:
Do we want to do this in this PR (changing bridge/js.ts into bridge/index.ts) ? the changes would span across multiple coins

From [libs/ledger-live-common/scripts/sync-families-dispatch.mjs](https://github.com/LedgerHQ/ledger-live/blob/develop/libs/ledger-live-common/scripts/sync-families-dispatch.mjs#L99) 

<img width="693" alt="image" src="https://github.com/user-attachments/assets/80ecb1aa-6183-4623-a2ce-6750c7983063">
(or we could handle both js.ts and index.ts but that could get confusing if we don't change them all)